flexmeasures/ui/templates/views/status.html

```diff
@@ -55,6 +58,91 @@ <h3>Data connectivity for sensors of {{ asset.name }}</h3>
                         {% endfor %}
                     </tbody>
                 </table>
+                <h3>Latest scheduling jobs of {{ asset.name }}</h3>
+                <table class="table table-striped table-responsive paginate nav-on-click">
+                    <thead>
+                        <tr>
+                            <th>Job id</th>
+                            <th>Asset id</th>
+                            <th>Sensor or asset</th>
+                            <th class="text-right no-sort">Status</th>
+                            <th class="hidden">URL</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for job_data in scheduling_job_data: %}
+                        <tr title="View data">
+                            <td>
+                                {{ job_data.job_id }}
+                            </td>
+                            <td>
```

Comment:
Request for another column: Let's add the time the job was queued here. Also, ordering the jobs by that attribute (e.g. in the view) would be good, earliest first.